templates/taxi/manufacturer_list.html

```diff
@@ -1,7 +1,8 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
+    <h1>Manufacturer List |
+      <a href="{% url 'taxi:manufacturer-create' %}">CREATE</a>
```

Comment:
and this world too

Reply:
![image](https://user-images.githubusercontent.com/111114742/200577006-ffd87772-9ef2-4ed7-80d7-b627abceb90b.png)